### ISSUE TITLE
At latest version problems when changing message_detail_level from basic to other

### ISSUE THREAD

**Issue by marcencov** [rackspace-cookbooks/chef-slack_handler#26]
Hi again.

I have problems after updating to latest 0.7.0 version. I've use web hooks method.

When changing message_detail_level from basic (with basic work without troubles) to other (elapsed, resources) I've receive error.

```
Running handlers:
[2016-05-06T13:20:29+03:00] WARN: Failed to send message to Slack: undefined local variable or method `updated_resources' for #<SlackHandlerUtil:0x0000000262dd68>
  - Chef::Handler::Slack
```

Also I try to change:

```
Valid options here are basic, elapsed, resources
default['chef_client']['handler']['slack']['message_detail_level'] = 'resources'
```

and for web hook too:
`default['chef_client']['handler']['slack']['webhooks']['webhook1']['message_detail_level'] = 'resources'.`

Can you check why? Previous 0.6.0 version work fine.

**Comment by cknowles** [comment]
Just having a quick look at it since I did the pull request that changed that bit of code. Will let you know shortly.

**Comment by cknowles** [comment]
For some reason I had my local kitchen test setup with the old `detail_level` setting. After updating it I caught this easily and fixed in my pull request.

**Comment by martinb3** [comment]
I've released 0.7.1 with the fixes from @c-knowles. Please let us know if that helped, @marchenkov :)

**Comment by marcencov** [comment]
Thank you. Helped.
